src/sentry/models/group.py

```diff
@@ -425,10 +425,8 @@ def update_group_status(
 
         modified_groups_list = []
         with transaction.atomic(router.db_for_write(Group)):
```

Comment:
Do we need the transaction? I feel like that was also added in the previous pr?

Reply:
It'll hold open a transaction while you select then update rows, probably better off not having it